### PR TITLE
Spelling corrections

### PR DESCRIPTION
**Pull request by mark-petersen**
This is my collection bin of spelling corrections.  They are in comments and RK4 timers.  Also, remove legacy CESM reference from comments!

### CODE REVIEW

**Comment by mark-petersen** [comment]
@sbrus89 and @xylar I think a quick visual inspection is enough. I will compile and test with the regression suite, and post those results.

File: src/core_ocean/shared/mpas_ocn_diagnostics.F [hunk 1]
```diff
@@ -2167,8 +2167,8 @@ subroutine ocn_validate_state(domain, timeLevel)!{{{
                call ocn_write_field_statistics(debugUnit, fieldName, minValue, maxValue)
             end if
 
-            ! Test snowFlow
-            fieldName = 'snowFlow'
+            ! Test snowFlux
+            fieldName = 'snowFlux'
```
Comment (xylar) [inline src/core_ocean/shared/mpas_ocn_diagnostics.F]: This one seems like an actual bug, not just a typo.

Comment (mark-petersen) [reply src/core_ocean/shared/mpas_ocn_diagnostics.F]: OK. See #175.

**Comment by mark-petersen** [comment]
Extracted bug to #175.  Thanks guys.

**Comment by mark-petersen** [comment]
Retested local merge with MPAS nightly regression suite, and on E3SM:
```
PASS PEM_Ln3.T62_oEC60to30v3wLI.GMPAS-DIB-IAF-ISMF.cori-knl_gnu RUN time=955
PASS PEM_Ln9.ne30_oECv3wLI.A_WCYCL1850-DIB-ISMF_CMIP6.cori-knl_gnu RUN time=665
PASS SMS_D.T62_oQU120_ais20.MPAS_LISIO_TEST.cori-knl_gnu RUN time=1114
PASS SMS_D_Ln9.T62_oQU120_ais20.MPAS_LISIO_TEST.cori-knl_intel RUN time=720
```

**Pull request merged**